profiles/schema.py

```diff
@@ -379,35 +380,35 @@ class Meta:
         AddressNode,
         description="Convenience field for the address which is marked as primary.",
     )
-    emails = DjangoFilterConnectionField(
+    emails = DjangoConnectionField(
         EmailNode, description="List of email addresses of the profile."
     )
-    phones = DjangoFilterConnectionField(
+    phones = DjangoConnectionField(
         PhoneNode, description="List of phone numbers of the profile."
     )
-    addresses = DjangoFilterConnectionField(
+    addresses = DjangoConnectionField(
         AddressNode, description="List of addresses of the profile."
     )
```

Comment:
@akikoskinen I'll also remove it from the `ContactNode` since all those three inherit from it, and it wouldn't make any difference to remove it from the child but not the parent (those are the only ones inheriting `ContactNode`).

Reply:
Good stuff.